packages/fast-usdc/src/exos/advancer.js

```
@@ -222,39 +216,30 @@ export const prepareAdvancerKit = (
       },
       transferHandler: {
         /**
-         * @param {undefined} result TODO confirm this is not a bigint (sequence)
+         * @param {unknown} result TODO confirm this is not a bigint (sequence)
          * @param {AdvancerVowCtx} ctx
          */
         onFulfilled(result, ctx) {
           const { notifyFacet } = this.state;
-          const { amount, destination, forwardingAddress, txHash } = ctx;
+          const { advanceAmount, destination, ...detail } = ctx;
           log(
             'Advance transfer fulfilled',
-            q({ amount, destination, result }).toString(),
-          );
-          notifyFacet.notifyAdvancingResult(
-            txHash,
-            forwardingAddress,
-            amount.value,
-            destination.value,
-            true,
+            q({ advanceAmount, destination, result }).toString(),
           );
+          try {
+            notifyFacet.notifyAdvancingResult({ destination, ...detail }, true);
+          } catch (err) {
+            log('@@@ call to notifyAdvancingResult', err);
+          }
```

When would this throw? We have different catch behavior in the `onRejected` below

---

Oops. @@@ means I wasn't finished with this.  Good catch. Will give it another think.

---

It was due to a bug; I saw:

```
Error#2: no advancing tx with {
  sender: 'noble1x0ydg69dh6fqvr27xjvp6maqmrldam6yfelqkd',
  amount: 105839999n
}
```

The thing is: if something throws in `onFulfilled`, the exception gets swallowed up. So I wonder if _all_ of our `onFulfilled` (and `onRejected`) methods need a try/catch.

---

There's a new issue about this:
 - #10576